p2p_wallet/Scenes/Main/Crypto/Components/Crypto Accounts/CryptoAccountsInteractor.swift

```diff
@@ -0,0 +1,132 @@
+import AnalyticsManager
+import Combine
+import Foundation
+import KeyAppBusiness
+import KeyAppKitCore
+import Resolver
+import Sell
+import SolanaSwift
+import SwiftyUserDefaults
+import Web3
+import Wormhole
+
+protocol CryptoAccountsInteractorProtocol {
+    var transferAccountsPublisher: AnyPublisher<[any RenderableAccount], Never> { get }
+    var primaryAccountsPublisher: AnyPublisher<[any RenderableAccount], Never> { get }
+    var hiddenAccountsPublisher: AnyPublisher<[any RenderableAccount], Never> { get }
+    var zeroBalanceTogglePublisher: AnyPublisher<Bool, Never> { get }
+    
+    func refreshServices() async
+    func updateFavorites(renderableAccount: RenderableSolanaAccount)
+}
+
+final class CryptoAccountsInteractor: CryptoAccountsInteractorProtocol {
+    
+    // MARK: - Dependencies
+    
+    @Injected private var analyticsManager: AnalyticsManager
+    @Injected private var solanaAccountsService: SolanaAccountsService
+    @Injected private var ethereumAccountsService: EthereumAccountsService
+    @Injected private var userActionService: UserActionService
+    @Injected private var favouriteAccountsStore: FavouriteAccountsDataSource
+    
+    // MARK: - Initialization
+    
+    init() {
+        bindZeroBalanceToggle()
+        bindAccounts()
+    }
+    
+    // MARK: - Properties
+    
+    private let cryptoAccountsAggregator = CryptoAccountsAggregator()
+    
+    private var subscriptions = Set<AnyCancellable>()
+    private var defaultsDisposables: [DefaultsDisposable] = []
+    
+    private var _transferAccountsPublisher = CurrentValueSubject<[any RenderableAccount], Never>([])
```

Comment:
This can be named transferAccountsSubject to distinguish with transferAccountPublisher.